.github/workflows-source/schedule-daily.yml

```diff
@@ -132,6 +132,36 @@ jobs:
           ZH2_DLL01_CSV_SECRETS: "${{ secrets.ZH2_DLL01_CSV_SECRETS }}"
           ZH2_FILE_SHARE_KEY: "${{ secrets.ZH2_FILE_SHARE_KEY }}"
 
+  fi-tests-nightly:
+    name: Bazel Test FI Nightly
+    <<: *dind-large-setup
+    timeout-minutes: 120
+    steps:
+      - <<: *checkout
+      - <<: *before-script
+      - name: Run FI Tests Nightly
+        id: bazel-test-all
+        uses: ./.github/actions/bazel-test-all/
+        with:
+          BAZEL_COMMAND: "test"
+          BAZEL_TARGETS: "//rs/rosetta-api/..."
+          BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
+          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=fi_tests_nightly --test_env=SSH_AUTH_SOCK"
+          HONEYCOMB_API_TOKEN: ${{ secrets.HONEYCOMB_API_TOKEN }}
+          SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      - name: Upload bazel-bep
```

Comment:
This is not strictly necessary unless it's useful to download bazel logs after.

Reply:
Thanks for the feedback! I'll keep it for now, and consider removing it later in case we don't find them useful.